src/commands/lambda/__tests__/uninstrument.test.ts

```diff
@@ -583,160 +548,16 @@ describe('lambda', () => {
           throw Error('Update failed!')
         })
 
-        const cli = makeCli()
-        const context = createMockContext()
         const functionARN = 'arn:aws:lambda:us-east-1:123456789012:function:lambda-hello-world'
-        const code = await cli.run(
-          ['lambda', 'uninstrument', '-f', functionARN, '--profile', 'SOME-AWS-PROFILE'],
-          context
-        )
-        const output = context.stdout.toString()
+        const {code, context} = await runCLI(['-f', functionARN, '--profile', 'SOME-AWS-PROFILE'])
         expect(code).toBe(1)
-        expect(output).toMatchInlineSnapshot(`
+        expect(context.stdout.toString()).toMatchInlineSnapshot(`
           "
           🐶 Uninstrumenting Lambda function
           [Error] Error: Couldn't set AWS profile credentials. Update failed!
           "
         `)
       })
-
-      test('prints which functions failed to uninstrument without aborting when at least one function was uninstrumented correctly', async () => {
```

Comment:
Moved this whole test to the [test of the same name](https://github.com/DataDog/datadog-ci/blob/0944aea9bb1f61ad2a05faaf6978a367ba61d6e0/src/commands/lambda/__tests__/instrument.test.ts#L1191) in `src/commands/lambda/__tests__/instrument.test.ts`, because it was using `cli.run()` with the `lambda instrument` command while it was in the `uninstrument.test.ts` file.

Reply:
I'm pretty sure this (and same issue in test below) is a typo and it's supposed to be using the uninstrument command. I'm working on the exact suggestion to give you now

Reply:
Possible indeed, thank you!

Reply:
Sounds good to me! Thanks again :grin: